core/unit_test/tstNeighborList.hpp

```diff
@@ -586,7 +586,7 @@ void testFirstNeighborParallelFor()
     ListType nlist( Cabana::slice<0>( aosoa ), 0, aosoa.size(), test_radius,
                     cell_size_ratio, grid_min, grid_max );
 #else
-    (void)cell_size_ratio;
+    std::ignore = cell_size_ratio;
     using device_type = TEST_MEMSPACE; // sigh...
```

Comment:
Yeah we need to fix the use of `Device` in our classes. This stuff was written when we thought that was the best design pattern.